timesketch/api/v1/resources/user.py

```diff
@@ -46,6 +47,84 @@ def get(self):
         """
         return self.to_json(User.query.all())
 
+    @login_required
+    def post(self):
+        """Handles POST request to the resource.
+
+        Returns:
+            User Object
+        """
+
+        if not current_user.admin:
+            abort(
+                HTTP_STATUS_CODE_FORBIDDEN,
+                "The user has no permissions to create other users.",
+            )
+        form = request.json
+        username = form.get("username", "")
+        password = form.get("password", "")
+
+        # Check provided username
+        if not username:
+            abort(
+                HTTP_STATUS_CODE_NOT_FOUND,
+                "No username provided, unable to create the user.",
+            )
+        if not isinstance(username, str):
+            abort(HTTP_STATUS_CODE_FORBIDDEN, "Username needs to be a string.")
+        # Check provided password
+        if not password:
+            abort(
+                HTTP_STATUS_CODE_NOT_FOUND,
+                "No password provided, unable to create the user.",
+            )
+        if not isinstance(password, str):
+            abort(HTTP_STATUS_CODE_FORBIDDEN, "Password needs to be a string.")
+
+        user = User.get_or_create(username=username, name=username)
+        user.set_password(plaintext=password)
+        # TODO: Take additional attributes of users into account
+        db_session.add(user)
+        db_session.commit()
+
+        user = User.get_or_create(username=username)
+
+        return_user = {
+            "id": user.id,
+            "username": user.username,
+            "name": user.name,
+            "email": user.email,
+            "active": user.active,
+            "admin": user.admin,
+        }
+
+        return jsonify({"objects": [return_user]})
```

Comment:
```suggestion
return self.to_json(user)
```
You have added the fields to the resource field mapping in __init__. You just need to call self.to_json here